docs/documentation/development/bindings/categories.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Categories
+
+Categories in Eclipse SmartHome are used to provide meta information about things channels, etc. UIs can use this information to render specific icons or provide a search functionality to for example filter all things for a certain category.
+
+## Thing Categories
+
+The thing type definition allows to specify a category. User interfaces can parse this category to get an idea how to render this thing. A binding can classify each thing into one of the existing categories. The list of all predefined categories can be found in our categories overview:
+
+| Category        | Description                                          |
+|-----------------|------------------------------------------------------|
+
+
+## Channel Categories
```

Comment:
As you have copied this list from an existing page, shouldn't you remove it from the old location (and merely link to the new page where appropriate)?